build/README.md

```diff
@@ -0,0 +1,19 @@
+# Lisk Build
+
+## Linux
+
+A Ubuntu Linux 16.04 LTS (base install) machine is assumed.
+
+### Prerequisistes
+
+The following packages must be installed to build releases with lisk-build:
+
+```
+sudo apt-get install autoconf build-essential git jq libtool moreutils python2.7 tcl8.5 wget zlib1g-dev
+```
+
+### How-To
+
+```
+make LISK_NET=main
```

Comment:
We use variable `LISK_NETWORK` all around the app, please use same here as well. Also use full network names.

Reply:
`LISK_NETWORK` would contain `mainnet` while `LISK_NET` is set to `main`; this is because `build.sh` requires `main` or `test` to be passed as parameter.